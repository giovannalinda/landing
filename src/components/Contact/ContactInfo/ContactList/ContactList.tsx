import { FiMapPin, FiMail, FiPhone } from 'react-icons/fi'

import * as S from './ContactList.styled'

export function ContactList() {
  return (
    <S.ContactList>
      <li>
        <S.Icon>
          <FiMapPin size={26} />
        </S.Icon>
        <small>
          House #5, Street Number #98,
          <br /> brasilia- 70000-000, Brazil.
        </small>
      </li>

      <li>
        <S.Icon>
          <FiMail size={26} />
        </S.Icon>
        <small>
          albert.design@gmail.com <br />
          albert.flores@gmail.com
        </small>
      </li>

      <li>
        <S.Icon>
          <FiPhone size={26} />
        </S.Icon>
        <small>
          +55 955 258 2699 <br /> +55 955 100 9449
        </small>
      </li>
    </S.ContactList>
  )
}
