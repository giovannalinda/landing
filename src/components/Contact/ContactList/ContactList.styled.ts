import styled from 'styled-components'
import { theme } from '~/styles'

export const ContactList = styled.ul`
  display: grid;
  grid-template-rows: repeat(3, 1fr);
  grid-gap: ${theme.space[5]};

  li {
    display: flex;
    align-items: center;
    flex-direction: row;
  }

  small {
    font-size: ${theme.font.sizes.large};
    margin-left: 2rem;
    line-height: 2.8rem;
  }
`

export const Icon = styled.div`
  background: #f0edff;
  border-radius: ${theme.borderRadius.rounded};
  width: 8rem;
  height: 8rem;
  display: flex;
  align-items: center;
  justify-content: center;

  svg {
    color: ${theme.colors.secondaryPink};
  }
`
