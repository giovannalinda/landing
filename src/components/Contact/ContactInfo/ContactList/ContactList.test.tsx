import { render } from '~/__helpers__/app-tests'
import { ContactList } from './ContactList'

describe('ContactList', () => {
  it('should match snapshot', () => {
    const { container } = render(<ContactList />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
