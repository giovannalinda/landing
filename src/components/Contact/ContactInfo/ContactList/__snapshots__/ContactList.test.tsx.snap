// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactList should match snapshot 1`] = `
<ul
  class="sc-bczRLJ igNxfJ"
>
  <li>
    <div
      class="sc-gsnTZi hRYDKG"
    >
      <svg
        fill="none"
        height="26"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="26"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21 10c0 7-9 13-9 13s-9-6-9-13a9 9 0 0 1 18 0z"
        />
        <circle
          cx="12"
          cy="10"
          r="3"
        />
      </svg>
    </div>
    <small>
      Diadema,
      <br />
       SÃ£o Paulo - SP, Brazil.
    </small>
  </li>
  <li>
    <div
      class="sc-gsnTZi hRYDKG"
    >
      <svg
        fill="none"
        height="26"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="26"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"
        />
        <polyline
          points="22,6 12,13 2,6"
        />
      </svg>
    </div>
    <small>
      eugiovannasouza@gmail.com 
      <br />
      oigiovannalinda@gmail.com
    </small>
  </li>
  <li>
    <div
      class="sc-gsnTZi hRYDKG"
    >
      <svg
        fill="none"
        height="26"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="26"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
        />
      </svg>
    </div>
    <small>
      +55 955 258 2699 
      <br />
       +55 955 100 9449
    </small>
  </li>
</ul>
`;
